web-ui/src/components/guides/PDLGuidesPanel.jsx

```diff
@@ -1,43 +1,41 @@
-import React, { useContext } from 'react';
+import { useContext, useState } from 'react';
+
 import { AppContext } from '../../context/AppContext';
-import PdfIcon from '@mui/icons-material/PictureAsPdf';
-import Card from '@mui/material/Card';
-import CardHeader from '@mui/material/CardHeader';
-import List from '@mui/material/List';
+import { selectCsrfToken, selectRoles, selectUserProfile } from '../../context/selectors.js';
+
+import { fetchDocumentsForRole, generate } from './GuidesPanel.jsx';
 import './GuidesPanel.css';
-import GuideLink from './GuideLink';
+
+const fallbackPdfs = [
+  {
+    id: '1',
+    name: 'Development Discussion Guide for PDLs',
+    url: '/pdfs/Development_Discussion_Guide_for_PDLs.pdf'
+  },
+  {
+    id: '2',
+    name: 'Expectations Discussion Guide for PDLs',
+    url: '/pdfs/Expectations_Discussion_Guide_for_PDLs.pdf'
+  },
+  {
+    id: '3',
+    name: 'Feedback Discussion Guide for PDLs',
+    url: '/pdfs/Feedback_Discussion_Guide_for_PDLs.pdf'
+  }
+];
 
 const PDLGuidesPanel = () => {
   const { state } = useContext(AppContext);
-  const { userProfile } = state;
-  const isPdl =
-    userProfile &&
-    userProfile.role &&
-    userProfile.role.length > 0 &&
-    userProfile.role.includes('PDL');
-
-  const pdlPDFs = [
-    {
-      name: 'Development Discussion Guide for PDLs'
-    },
-    {
-      name: 'Expectations Discussion Guide for PDLs'
-    },
-    {
-      name: 'Feedback Discussion Guide for PDLs'
-    }
-  ];
+  const csrf = selectCsrfToken(state);
+  const isPdl = selectUserProfile(state)?.role?.includes('PDL');
 
-  return isPdl ? (
-    <Card>
-      <CardHeader avatar={<PdfIcon />} title="Development Lead Guides" />
-      <List dense>
-        {pdlPDFs.map(pdlPDF => (
-          <GuideLink key={pdlPDF.name} name={pdlPDF.name} />
-        ))}
-      </List>
-    </Card>
-  ) : null;
+  if (isPdl) {
+    const [documents, setDocuments] = useState([]);
```

Comment:
React won't allow conditional use of `useState` like this.

Reply:
How does it not being allowed show itself? In my tests, it works 🤔

Reply:
Were there any error messages in the terminal window where you ran "yarn start"?
I've never seen React fail to flag this.
The fix is to move line 33 to before line 32 so it's not conditional.

Reply:
Nope, the only error I see is

```
09:30:58 [vite] hmr invalidate /src/components/guides/GuidesPanel.jsx Could not Fast Refresh ("fetchDocumentsForRole" export is incompatible). Learn more at https://github.com/vitejs/vite-plugin-react-swc#consistent-components-exports
```

However, it does refresh when running with `yarn start`...

No errors about this being inside the conditional... did you see any?

[I've moved it out of the conditional anyway](https://github.com/objectcomputing/check-ins/pull/2522/commits/ebc7cded903d9e7f9d52c7e04672f22ffef4584e) 😄